libr/debug/p/native/linux/linux_debug.c

```diff
@@ -20,7 +20,11 @@ const char *linux_reg_profile (RDebug *dbg) {
 #elif __arm64__ || __aarch64__
 #include "reg/linux-arm64.h"
 #elif __MIPS__ || __mips__
+	if ((dbg->bits & R_SYS_BITS_32) && (dbg->bp->endian == true)) {
```

Comment:
uhm endian is 0 or 1.. because it may be 2 for middle endian. so better just do dbg->bp->endian == 1)